build.sh

```diff
@@ -2,30 +2,43 @@
 
 set -e
 
-cd "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+if [ -z "$1" ]; then
+  echo "Usage: build.sh destdir"
+  exit 1
+fi
+
+NVML_SRC_DIR=`dirname $(readlink -f $0)`
+NVML_BUILD_DIR=$(realpath "$1")"/wine-nvml"
+
+if [ -e "$NVML_BUILD_DIR" ]; then
+  echo "Build directory $NVML_BUILD_DIR already exists"
+  exit 1
+fi
 
 git submodule update --init
 
-meson setup \
-    --prefix /usr \
-    --libdir lib \
-    --buildtype release \
-    --cross-file build-wine64.txt \
-    build64 .
-
-ninja -C build64
-
-meson setup \
-    --prefix /usr \
-    --libdir lib32 \
-    --buildtype release \
-    --cross-file build-wine32.txt \
-    build32 .
-
-ninja -C build32
-
-if [[ "${1}" == --install ]]
-then
-    ninja -C build64 install
-    ninja -C build32 install
-fi
\ No newline at end of file
+function build_arch {
+  export WINEARCH="win$1"
```

Comment:
Is this necessary? We are calling into `winegcc` and not `wine`, so it shouldn't make any difference.